app/view/panel/NumericAttributeSlider.js

```diff
@@ -0,0 +1,132 @@
+ * associated layers.
+ *
+ * @class CpsiMapview.view.panel.NumericAttributeSlider
+ */
+Ext.define('CpsiMapview.view.panel.NumericAttributeSlider', {
+    extend: 'Ext.panel.Panel',
+    xtype: 'cmv_numericattributeslider',
+
+    requires: [
+        'Ext.form.field.Checkbox',
+        'Ext.slider.Multi',
+        'CpsiMapview.controller.panel.NumericAttributeSlider'
+    ],
+
+    controller: 'cmv_numericattributeslider',
+
+    layout: 'hbox',
+
+    config: {
+        minValue: 0,
+        maxValue: 15,
+        currLowerValue: 5,
+        currUpperValue: 11,
+        increment: 0.5,
+        sliderLabel: 'Num. attribute',
+        checkboxLabel: 'active?',
+        numericField: null, // required, or better at the layer level?
+        enabled: true
+    },
+
+    /**
+     * The CSS class of the slider
+     */
+    cls: 'cpsi-numeric-attribute-slider',
+
+    /**
+     * Construct an instance of the numeric attribute slider. We need to take
+     * care of the correct handling of the inherited disabled configuration.
+     *
+     * @param {object} cfg The configuration object.
+     */
+    constructor: function(cfg) {
+        var cfgEnabled = cfg.enabled;
+        var cfgDisabled = cfg.disabled;
+        delete cfg.enabled;
+        delete cfg.disabled;
+
+        var hasPassedCfgEnabled = Ext.isDefined(cfgEnabled);
+        var hasPassedCfgDisabled = Ext.isDefined(cfgDisabled);
+
+        var finalEnabled = cfgEnabled; // either undefined or user defined
+
+        if (hasPassedCfgEnabled && hasPassedCfgDisabled) {
+            // user passed both, ensure they make sense or warn
+            if (cfgEnabled !== !cfgDisabled) { // semantically not same
+                Ext.Logger.warn('Conflicting disabled/enabled configs found. '
+                    + 'NumericAttributeSlider will be using "enabled:'
+                    + finalEnabled + '".');
+            }
+        } else if (hasPassedCfgDisabled) {
+            // user passed only disabled, warn and switch to enabled
+            finalEnabled = !cfgDisabled;
+            Ext.Logger.info('Config option "disabled=' + cfgDisabled + '" passed,'
+                + ' transforming to option "enabled=' + finalEnabled + '".');
+        } else if (!hasPassedCfgEnabled) {
+            // reinitiate the default of this class
+            var clazz = CpsiMapview.view.panel.NumericAttributeSlider;
+            finalEnabled = clazz.prototype.config.enabled;
+        }
+
+        cfg.enabled = finalEnabled;
+
+        this.callParent([cfg]);
+    },
+
+    /**
+     * Initializes the numeric attribute slider.
+     */
+    initComponent: function () {
+        var me = this;
+        var values = [me.getCurrLowerValue(), me.getCurrUpperValue()];
+        me.items = [{
+            xtype: 'multislider',
+            name: 'slider',
+            labelAlign: 'right',
+            listeners: {
+                afterrender: 'initSlider',
+                changecomplete: 'applySliderEffects'
```

Comment:
Do I get it right that the `applySliderEffects` is always bound to `changecomplete` but the slider gets disabled in case the checkbox is unchecked?

Reply:
Yes. the slider gets disabled according to the checkbox state and depending on the state we also apply / revert the effects of the thumbs. On might argue that disabling this parent component would completely disable the checkbox as well. This can be changed if we want. Does that answer your question?

Reply:
Yes, this answers my question. All fine from my side :+1: